src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/start-free-scan/StartFreeScanView.tsx

```diff
@@ -50,8 +50,11 @@ export function StartFreeScanView(props: Props) {
           <p>
             {l10n.getString(
               "fix-flow-data-broker-profiles-start-free-scan-content-p2"
-            )}
-            <a href="#">
```

Comment:
Unrelated, but this got me curious&hellip;
Looks like we have two more `href="#"` in the codebase which may or may not be suspicious (or dead code). 🤷 

```sh
git rev-parse --short HEAD # ef35b490a
git grep -n "href=[\"']#[\"']" | cat

  src/app/(nextjs_migration)/(authenticated)/user/settings/page.tsx:35:      <a className="js-settings-resend-email" data-email-id={email.id} href="#">
  src/views/partials/settings.js:13:  <a class='js-settings-resend-email' data-email-id='${email.id}' href='#'>
```